src/less/toast-dialog/base/toast-dialog.less

```diff
@@ -144,3 +144,9 @@ button.toast-dialog__close {
         margin: 16px 24px 24px;
     }
 }
+
+[dir="rtl"] {
+    .btn.btn--primary {
```

Comment:
Nitpicking, but is the chained selector necessary? We try and avoid them if we can.

Reply:
probably not, I was just trying to find things that worked. 

I'll adjust it